### PR TITLE
Debounce search input

### PR DESCRIPTION
**Pull request by mikefrancis**
Prettier made a load of noise too, sorry about that. I think it had been saved with different formatting than in `.prettierrc` - I would highly recommend turning `Format on save` on in VS Code 💯

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "gatsby-source-filesystem": "^2.0.32",
     "gatsby-transformer-remark": "^2.3.12",
     "gatsby-transformer-sharp": "^2.1.18",
+    "lodash": "^4.17.11",
```
Comment (lukewhitehouse) [inline package.json]: Probably best to include [`lodash.debounce`](https://www.npmjs.com/package/lodash.debounce) instead for speedier installs.

Comment (lukewhitehouse) [reply package.json]: also forgot to say on here, but thanks for the PR Mikey! ❤️

Comment (mikefrancis) [reply package.json]: Nice, didn't know about `lodash.debounce`. Guess it's perfect if you just need that function 😄

**Review by lukewhitehouse — APPROVED** [review]
You the bestest 🎉

**Pull request merged**